src/app/services/manage-people/manage-people.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+
+import { IUser } from 'interfaces/user';
+
+import { AgeVerificationService } from 'services/age-verification/age-verification.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ManagePeopleService {
+  constructor(private readonly ageVerification: AgeVerificationService) {}
+
+  olderThanLegalAge(users: IUser[] = []): IUser[] {
```

Comment:
Add 2 tests for it, first one when you do not pass users, another one when u pass parameters.